### PR TITLE
Bonnie 157 fixture export task

### PR DESCRIPTION
**Pull request by alexanderelliott121**
Adds a fixture export task.

### CODE REVIEW

**Comment by alexanderelliott121** [comment]
Chris Tohnline has brought up the subject of error reporting.
At the moment, failure utilizes the native errors that will be thrown by Mongoid when a search returns empty results.

File: lib/tasks/generate_fixtures.rake [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        test_name["\."] = "_"
+      end
+      
+      user = User.find_by email: args[:user_email]
+      if (args[:use_hqmf] == true)
+        measure = user.measures.find_by(hqmf_id: args[:cms_id])
+      else
+        measure = user.measures.find_by(cms_id: args[:cms_id])
+      end
+      patient = Record.find_by(user: user, first: args[:patient_first_name], last: args[:patient_last_name])
```
Comment (mulcahyk) [inline lib/tasks/generate_fixtures.rake:28]: You are searching by the whole `user` object here.  I think that you want to be searching by just the user's id so like this: `user: user.id`.

File: lib/tasks/generate_fixtures.rake [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      
+      fixtures_directory = File.join("test", "fixtures")
+      output_directory = File.join(fixtures_directory, args[:test_set], test_name)
+      Dir.mkdir(output_directory) unless Dir.exists? output_directory
```
Comment (mulcahyk) [inline lib/tasks/generate_fixtures.rake]: When I run I get this: `Errno::ENOENT: No such file or directory @ dir_s_mkdir - test/fixtures/measure_history/fake`.  It doesn't seem to matter what I pass in for the `test_set` and `test_name` parameters, I still get some kind of error when creating the directory.

File: lib/tasks/generate_fixtures.rake (hunk 1 above)
Comment (alexanderelliott121) [reply lib/tasks/generate_fixtures.rake:28]: I believe that the user field invokes the scope: by_user in bonnie's extension of record.
I could look into this more deeply, but the mongoid documentation I've seen so far ha been unhelpful on this front.

File: lib/tasks/generate_fixtures.rake (hunk 2 above)
Comment (alexanderelliott121) [reply lib/tasks/generate_fixtures.rake]: I've added support for recursive file creation.

**Pull request closed without merging**